packages/@react-native-windows/tester/src/js/examples/View/ViewExample.windows.js

```diff
@@ -34,7 +33,8 @@ class ViewBorderStyleExample extends React.Component<
     return (
       <TouchableWithoutFeedback
         testID="border-style-button"
-        onPress={this._handlePress}>
+        onPress={this._handlePress}
+        accessible>
```

Comment:
Are we going to upstream this? -- Should TouchableWithoutFeedback be accessible by default? -- Are we sure it isn't on the other platforms?

Reply:
Do we need an accessibilityLabel to be set since we are making it accessible?

Reply:
I'll verify this.